ledger_app.toml

```diff
@@ -5,3 +5,8 @@ devices = ["nanos", "nanox", "nanos+", "stax"]
 
 [tests]
 pytest_directory = "./tests/"
+
+[tests.dependencies]
+testing_with_prod = [
+    {url = "https://github.com/LedgerHQ/app-ethereum", ref = "fbe/add_use_cases_to_eth_manifest", use_case = "set_test_keys"}
+]
```

Comment:
I guess that we will replace this ""fbe/add_use_cases_to_eth_manifest"" soon ?

Reply:
Yes indeed. This is just for test purposes. We will most likely put "develop" in the end.